client/state/preferences/selectors.js

```diff
@@ -36,6 +36,26 @@ export function getPreference( state, key ) {
 	);
 }
 
+export function getMediaScalePreference( state, key, isMobile ) {
+	const SCALE_TOUCH_GRID = 0.323;
+	const mediaScale = get(
+		find(
+			[
+				state.preferences?.localValues,
+				state.preferences?.remoteValues,
+				DEFAULT_PREFERENCE_VALUES,
+			],
+			( source ) => has( source, key )
+		),
+		key,
+		null
+	);
```

Comment:
This should call the `getPreference()` selector instead of duplicating its code.